p2p/datasets_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestRequestDatasetsList(t *testing.T) {
```

Comment:
This test is slated 2nd for refactoring when we overhaul the p2p test network setup!